reporting/impl/src/main/resources/reports/resources/css/tech-report-punchcard.css

```diff
@@ -106,3 +125,81 @@ table.technologiesPunchCard thead,
 table.technologiesPunchCard tbody tr {
     display: table;
 }
+
+/* Tooltip container */
+.table-tooltip {
+    position: relative;
+}
+
+/* Tooltip text */
+.table-tooltip .table-tooltiptext {
+    visibility: hidden;
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+
+    position: absolute;
+    z-index: 1;
+
+    font-size: 14pt;
+    padding: 5px;
+
+    /*left tooltip*/
+    top: 10px;
+    right: 85%;
+
+    opacity: 0;
+    transition: opacity 1s;
+}
+
+.table-tooltip:hover .table-tooltiptext {
+    visibility: visible;
+    opacity: 1;
```

Comment:
This should be changes to `opacity: 0.9;` to consistency with all the other tooltips in the report.